### PR TITLE
publish to crates.io

### PR DESCRIPTION
**Pull request by emturner**
Prepare crates for release to crates.io

### CODE REVIEW

File: tezos-encoding/Cargo.toml [hunk 1]
```diff
@@ -1,9 +1,14 @@
 [package]
 name = "tezos_encoding"
```
Comment (zcabter) [inline tezos-encoding/Cargo.toml:2]: wdyt about renaming this crate to tezos-data-encoding?

File: crypto/README.md [hunk 2]
```diff
@@ -1,4 +1,12 @@
-Crypto
+Tezos crypto
 ===========
 
-Component contains cryptographic algorithms for encryption and decryption of messages
\ No newline at end of file
+Component contains cryptographic algorithms for hashing, signing & signature verification, with a slant towards
+those used by `tezos` - and in particular the `Kernel SDK` for smart rollups.
```
Comment (zcabter) [inline crypto/README.md:5]: If possible. are there links to docs for tezos and kernel sdk that we could add?

**Review by zcabter — APPROVED** [review]
LGTM, only minor comments.

File: tezos-encoding/Cargo.toml (hunk 1 above)
Comment (emturner) [reply tezos-encoding/Cargo.toml:2]: lets do it!

File: crypto/README.md (hunk 2 above)
Comment (emturner) [reply crypto/README.md:5]: have done for `tezos` - but for now there's nothing really linkable for the SDK - will update once we have some proper SDK docs

**Review by zcabter — APPROVED** [review]

**Pull request merged**